src/lib/plugin.js

```diff
@@ -74,6 +106,14 @@ class FiveBellsLedger extends EventEmitter2 {
     this.account = options.account
     this.username = options.username
 
+    // optional: use account@ledger.example in order to authenticate
+    // instead of using account URI + username
+    this.identifier = options.identifier
```

Comment:
the identifier is never used as user/password/account, it just resolves to them. `credentials` is for separating the authorization from the account you want to subscribe to